js/client-chat.js

```diff
@@ -1237,6 +1237,13 @@
 					this.$chat.append('<div class="notice">' + Tools.sanitizeHTML(row.slice(1).join('|')) + '</div>');
 					break;
 
+				case 'notify':
+					if (!Tools.prefs('mute') && Tools.prefs('notifvolume')) {
+						soundManager.getSoundById('notif').setVolume(Tools.prefs('notifvolume')).play();
+					}
+					this.notifyOnce(row[1], "\"" + row.slice(2).join('|') + "\"", 'highlight');
```

Comment:
Highlights have quotes because the message is from another user, not from PS itself. Messages from PS itself do not have quotes.

Declares, on the other hand, are intentionally designed to look like they're from PS itself.